### ISSUE TITLE
[Documentation] How to create an additional account

### ISSUE THREAD

**Issue by fuzeman** [trakt-for-plex/configuration#1]
#### 1. Login to the configuration interface

**1a.** Open http://trakt-for-plex.github.io/configuration

![](http://i.imgur.com/HRM4zXR.gif)

**1b.** Login with your plex.tv account credentials

--------------------------------------------

#### 2. Connect to your server

**2a.** Select your server

![](http://i.imgur.com/UHGvYkT.gif)

--------------------------------------------

#### 3. Create an account

**3a.** Open the "Accounts" configuration page *(http://trakt-for-plex.github.io/configuration/#/configuration/accounts)*

**3b.** Create a new account

![](http://i.imgur.com/61eagrh.gif)

**3c.** Authenticate the account

*(visit https://trakt.tv/pin/478 for an authentication pin for your trakt.tv account)*

![](http://i.imgur.com/XDv7tNM.gif)

--------------------------------------------

#### 4. Setup rules

**4a.** Open the "Rules" configuration page *(http://trakt-for-plex.github.io/configuration/#/configuration/rules)*

**4b.** Adjust the activity rules to automatically map plex users to their trakt.tv account

![](http://i.imgur.com/VNpLICO.gif)

--------------------------------------------

#### 5. Adjust global filters

**5a** Open the plugin preferences in Plex/Web

**5b.** Reset your global filters to allow activity from other users

![](http://i.imgur.com/Po182zu.gif)

--------------------------------------------

Happy to answer any questions below, let me know if you find any bugs :smiley:

**Comment by saitoh183** [comment]
Mapping is automatic? If I want to map each user to there own trakt account do I use User Rules and set account to the user's plex account or trakt account? And Name will be the other?

**Comment by saitoh183** [comment]
Setting up another User, Do i need a authentication Pin for that user as well? if so It wont Save

**Comment by fuzeman** [comment]
> Mapping is automatic? If I want to map each user to there own trakt account do I use User Rules and set account to the user's plex account or trakt account? And Name will be the other?

Yeah, if you have a "Map" rule added it will automatically match plex users to their trakt accounts (as defined in the "Accounts" pages). You only need to manually define user rules if you want to have multiple plex users scrobble to a single trakt account.

> Setting up another User, Do i need a authentication Pin for that user as well? if so It wont Save

Yes, the authentication pin is specific to the trakt.tv account.

Attempting to link two accounts to a single trakt.tv account will silently reject the save, did you open the https://trakt.tv/pin/478 URL after logging into a new trakt account?

**Comment by saitoh183** [comment]
Yes i did login to the other users trakt and then used the link.. I will try again again.

**Comment by fuzeman** [comment]
Upload the plugin log files if it fails again, see https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log_Files for the location of them.

**Comment by saitoh183** [comment]
Ok it worked this time...dont know what went wrong. Im gussing this doesnt worked with managed users?

**Comment by saitoh183** [comment]
Recommendation: 

put the  https://trakt.tv/pin/478 URL in the title area next to OAuth in the Trakt config section that way we dont have to go searching for it when we need to add a new user

**Comment by saitoh183** [comment]
Deleting Account is not working...nothing happens when i try it on a user account

**Comment by saitoh183** [comment]
In IE11 missing content:

![image](https://cloud.githubusercontent.com/assets/1534197/8830323/a70af230-3069-11e5-8c4b-b19211375011.png)

**Comment by saitoh183** [comment]
You should move the Doc to the Wiki section and add the link to the wiki in the support area on the configuration page

**Comment by fuzeman** [comment]
> put the https://trakt.tv/pin/478 URL in the title area next to OAuth in the Trakt config section that way we dont have to go searching for it when we need to add a new user

Forgot about this, will add this to my to-do list.

> You should move the Doc to the Wiki section and add the link to the wiki in the support area on the configuration page

Yeah, I'm planning on moving this into the wiki soon.

**Comment by MaartenMol96** [comment]
I seem to have a little issue. Set it up like the above, my own account is perfect. However, my friends accounts are scrobbling but not syncing the watched status. 

Does anyone have a clue how I could fix this?

**Comment by bbaraniec** [comment]
I can't authenticate 2nd user to Plex. I've set up Plex Home.

**Comment by fuzeman** [comment]
Just pushed an update ([v1.1.1](https://github.com/trakt-for-plex/configuration/releases/tag/1.1.1)) that supports creating accounts for Plex Home users *(this includes managed users)*.

On the "Plex" authentication tab change to the "Home" mode:
![](http://i.imgur.com/UqavOZ9.png)

**Comment by Mierzen** [comment]
Can it be set up so that if media is played on a certain device, there should be scrobbled to more than one account?

**Comment by fuzeman** [comment]
@Mierzen Interesting idea, have posted the feature here: https://github.com/trakt/Plex-Trakt-Scrobbler/issues/270

Not sure when this feature can be implemented, might be awhile away. Subscribe to that issue so you can get emails on the status of it :smiley:

**Comment by EldonMcGuinness** [comment]
I am hoping to have more than one account sync to a single trakt account, not just scrobble but actually be able to sync the play status. Is this possible?

**Comment by saitoh183** [comment]
@EldonMcGuinness 

yes you can...each account can scrobble or sync

**Comment by EldonMcGuinness** [comment]
@saitoh183 Perhaps I am missing something, can you elaborate as to the rules needed or what needs to happen? These are home accounts if that makes a difference.

I have the accounts created in plex and on the config page, tried putting:
* map -> eldon.mcguinness
* adults -> eldon.mcguinness

but when I try to trigger a sync in the trakt channel I get an oauth error. The eldon.mcguinness account  has a green check for plex and trakt on the config page, but adults will not get a green check for trakt, I'm assuming that is just the way it is, no?

**Comment by saitoh183** [comment]
Each Plex account associated with a trakt account

In Rules you you set it to Account = Map and Name =Any

**Comment by EldonMcGuinness** [comment]
@saitoh, what I'm trying to do is have 2 accounts sync with 1 trakt account.

eldon.mcguinness -> eldon.mcguinness
Adults -> eldon.mcguinness

What I'm trying to achieve here is to keep them in sync.

**Comment by saitoh183** [comment]
have you tried User Rules Map/eldon.mcguinness

**Comment by EldonMcGuinness** [comment]
yup, still the same.

**Comment by saitoh183** [comment]
i remembered seeing that question on the forum...and i found it...it is not possible...

https://forums.plex.tv/discussion/comment/1038803/#Comment_1038803

> Currently you can only send real-time watching activity from multiple plex accounts into one trakt account (via activity filters/rules).
> Syncing multiple plex accounts into one trakt account might be possible to implement, will have a look when I've got some free time.

**Comment by EldonMcGuinness** [comment]
bummer. Was hoping to allow my son to sync down the same data so when we watch his shows on the adults profile it could also be marked in his as watched too.

Perhaps it will be something that could happen in the future. Thanks for the assist.

**Comment by fuzeman** [comment]
I'll be looking at implementing some more advanced scrobbling/syncing options in future releases, currently focusing on getting the current release 100% stable first :smile:

**Comment by matrixagent** [comment]
Hi @fuzeman, I think I already asked that question a year ago or so, but I can't find our conversation and don't remember: Is it possible to run the configuration on my own server, and/or will my initial setup continue to work should the plex-for-trakt.github.io domain vanish for whatever reason? After having been burnt a few times too often by disappearing services, I try to minimize depending on external factors as much as possible. :wink: And thanks again for all your hard work, will definitely make another big donation after upgrading to 1.0!

**Comment by fuzeman** [comment]
@matrixagent Yeah, I can remember you asking this question awhile ago :smile:

The configuration website connects directly to the plugin running on your server to retrieve/update settings, so if the site disappears everything will continue working as usual.

As the site is hosted on GitHub's servers *(with [GitHub pages](https://pages.github.com/))* I doubt it would disappear, it might be unreachable for awhile if there is an outage at GitHub though..

**Comment by matrixagent** [comment]
@fuzeman Is it possible to map one Plex user to multiple trakt users? I'd like to set up a managed user that scrobbles for both me and my girlfriend. Both our individual Plex and trakt accounts are set up and work fine, but when we watch something together I'd rather select the "Both" Plex user and be done with it instead of one of us having to manually scrobble.

**Comment by fuzeman** [comment]
@matrixagent Currently the plugin doesn't support that. I would like to support more complex scrobbling/syncing configurations but it will likely be awhile before work can be started on this.

**Comment by matrixagent** [comment]
Good things come to those who wait, just like this fantastic new version. :)

**Comment by StefanFourie** [comment]
@fuzeman I think I might be missing something, but I cannot get my watched history onto the trakt profile.. I followed all the steps above, except for step 5 (cannot find it).

Any advice?

**Comment by infinitepi** [comment]
I know you've answered this one already, but i'd like to put in another plug for allowing multiple plex users to use 1 trakt account.

i have a plex home and have separate users for my daughter and our livingroom HTPC (for PlexPy tracking purposes mostly, i'm a data geek).  my main plex account is the one i use remotely and on the HTPC (will transition to Livingroom account when i can get it in sync).  Now i'm trying to get the watch statuses 'copied over' to the livingroom account, and going forward want them in sync.

Also, i almost broke my mouse getting pissed that the trakt account wouldn't save for my second user on the config website, till i read that it was 'silently failing'.  a visible error would have been nice....  ;)

thanks for all your work on this.

**Comment by fuzeman** [comment]
@StefanFourie 
> I think I might be missing something, but I cannot get my watched history onto the trakt profile.. 

Could you open a new issue at https://github.com/trakt/Plex-Trakt-Scrobbler with the following details:
 - Plugin version (displayed at: ```Plex/Web -> Channels -> trakt -> About```)
 - Plex client(s) used for watching media
 - Is scrobbling working *(activity displayed on your Trakt.tv profile while you are watching it)*? or is this just an issue with syncing *(pushing previously watched items to your Trakt.tv profile)*?
 - After running a "Full" sync, attach the latest plugin [log file](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files)

> I followed all the steps above, except for step 5 (cannot find it).

Can you see the "trakt" plugin listed under "Channels" in Plex/Web? The "Global Filter" options should be listed under the settings there.

> Also, 'pull' and 'push' is not really clear.. Is it pulling/pushing to or from Plex or Trakt?

**Pull:** Pulls data from your Trakt.tv profile into Plex
**Push:** Pushes data from Plex to your Trakt.tv profile

-----------------------------------------------

@infinitepi 
I'll probably look at that feature after support for AniDB metadata agents reaches the beta branch *(likely in April)*.

> Also, i almost broke my mouse getting pissed that the trakt account wouldn't save for my second user on the config website, till i read that it was 'silently failing'. a visible error would have been nice.... ;)

Yeah, the configuration site could do with some polishing in a few places. I'll make sure that issue is fixed in a future release :smile:

**Comment by valantislevas** [comment]
hello may i ask something is great service and i use it.. i already i put 3 friends to my server and now i change my server is possible to move the current users members from server to other server? without using again autherication etc?

thanks for this

**Comment by valantislevas** [comment]
also another question i add sometimes user and i make auth with trakt is pin is ok when i try with plex with basic or with code is ok it says and picture is shows up i save it isnt Ticked green near and it shows up again Code for link with plex :octopus:

**Comment by kaiguy** [comment]
I've noticed that all of my shared users scrobble to my personal trakt account.  I don't have a desire for my users to scrobble at this time, so what's the best way to keep it just to my own account.  Should I perform the "map" action as shown above, or just filter on my Plex.tv account name?  Thanks!

**Comment by infinitepi** [comment]
I say filter, thats how i solved it

**Comment by fuzeman** [comment]
> I've noticed that all of my shared users scrobble to my personal trakt account. I don't have a desire for my users to scrobble at this time, so what's the best way to keep it just to my own account? Should I perform the "map" action as shown above, or just filter on my Plex.tv account name? Thanks!

Either method will work, just use what is easier for you.

**Comment by mbnn** [comment]
Any idea why http://trakt-for-plex.github.io/ takes ages to load when i click on my server? 
I have this every single time, it takes upto 4 minutes before I see the settings.

**Comment by fuzeman** [comment]
@mbnn Could you do the following:
 - Open http://trakt-for-plex.github.io/configuration in your browser
 - Open the browser console (`Ctrl + Shift + K` on Firefox, or `Ctrl + Shift + J` on Chrome)
 - Connect to your server again
 - Paste the entries displayed in the console here

I think this has been reported once before, not sure if a solution was found though..

**Comment by mbnn** [comment]
Here is the output:

Registered interface "/api/home"
vendor.859d191d.js:20 Registered interface "/api/home/users"
vendor.859d191d.js:20 Registered interface "/api/invites"
vendor.859d191d.js:20 Registered interface "/api"
vendor.859d191d.js:20 Registered interface "/pms/:"
vendor.859d191d.js:20 Registered interface "/pms/playlists"
vendor.859d191d.js:20 Registered interface "/pms/social"
vendor.859d191d.js:20 Registered interface "/devices"
vendor.859d191d.js:20 Registered interface "/:"
vendor.859d191d.js:20 Registered interface "/pins"
vendor.859d191d.js:20 Registered interface ""
vendor.859d191d.js:20 Registered interface "/users"
vendor.859d191d.js:20 Registered interface "/:/plugins/*/messaging"
vendor.859d191d.js:20 Registered interface ""
vendor.859d191d.js:20 Exposed interface "/api/home"
vendor.859d191d.js:20 Exposed interface "/api/home/users"
vendor.859d191d.js:20 Exposed interface "/api/invites"
vendor.859d191d.js:20 Exposed interface "/api"
vendor.859d191d.js:20 Exposed interface "/pms/:"
vendor.859d191d.js:20 Exposed interface "/pms/playlists"
vendor.859d191d.js:20 Exposed interface "/pms/social"
vendor.859d191d.js:20 Exposed interface "/devices"
vendor.859d191d.js:20 Exposed interface "/:"
vendor.859d191d.js:20 Exposed interface "/pins"
vendor.859d191d.js:20 Exposed interface ""
vendor.859d191d.js:20 Exposed interface "/users"
vendor.859d191d.js:20 Exposed root method "devices"
vendor.859d191d.js:20 Exposed root method "pins"
vendor.859d191d.js:20 plex initialized
scripts.4ef01a59.js:1 authenticated
scripts.4ef01a59.js:2 [XXXXXXXXXXXXXXXXXXXXX] Testing 3 connections
vendor.859d191d.js:20 Exposed interface "/:/plugins/*/messaging"
vendor.859d191d.js:20 Exposed interface ""
vendor.859d191d.js:20 Exposed root method "details"
vendor.859d191d.js:20 Exposed root method "identity"
scripts.4ef01a59.js:2 [XXXXXXXXXXXXXXXXXXXXX] Testing connection: https://172-16-6-2.2d7f2039a5a840f5b9a6f5c5e7e7ad17.plex.direct:32400
vendor.859d191d.js:19 OPTIONS https://172-16-6-2.2d7f2039a5a840f5b9a6f5c5e7e7ad17.plex.direct:32400/identity net::ERR_CONNECTION_TIMED_OUT(anonymous function) @ vendor.859d191d.js:19
vendor.859d191d.js:20 Exposed interface "/:/plugins/*/messaging"
vendor.859d191d.js:20 Exposed interface ""
vendor.859d191d.js:20 Exposed root method "details"
vendor.859d191d.js:20 Exposed root method "identity"
scripts.4ef01a59.js:2 [XXXXXXXXXXXXXXXXXXXXX] Testing connection: https://172-16-5-2.2d7f2039a5a840f5b9a6f5c5e7e7ad17.plex.direct:32400

I have no idea what that 172.16 ip is, but its definitely not our ip.

**Comment by fuzeman** [comment]
@mbnn Just pushed an update to the configuration site ([v1.2.1](https://github.com/trakt-for-plex/configuration/releases/tag/1.2.1)), let me know if this reduces connection times.

> I have no idea what that 172.16 ip is, but its definitely not our ip. Actually that 172.16 is our 2nd, 3rd IP on the LAN adapter for internal data transfer. No idea why it would try to connect to that though?

Plex Media Server advertises the IP Addresses of all your network interfaces, the configuration site needs to cycle through the available IP addresses for your server and find a valid one.

**Comment by mbnn** [comment]
This seems to have fixed it, now its 4 seconds :) Thank you!

**Comment by skycryer** [comment]
I tried to setup my accounts but i got an error while selecting the correct server. I have to plex server with latest trakt and plex software. But i get Http Error 404 when selection a server. Anyone else with this problem or some help maybe? thx

**Comment by jadroher** [comment]
Hello @fuzeman 

I have my plex main account synced with the trakt that seems to be working fine. But I wanted to use one of the plex user managed accounts instead of the main one. So following your instructions I change the account configuration (http://trakt-for-plex.github.io/configuration/#/configuration/accounts?id=1) and selected "home" when linking the plex and selected the plex user I wanted to be in sync. In the rules section I have Nothing in the client rules and the user rules set to Map as you suggest. But I see that it still syncs the main account.  What I am doing wrong? Do I need to reset something in my trakt account?

Thanks
Juanjo

**Comment by fuzeman** [comment]
@jadroher The first account on the configuration site will always sync to the administrator of your Plex Media Server, the site really needs to be updated to detail this clearly...

Try the following:

1. Disconnect your Trakt.tv profile from the administrator (first) account
2. Create a new account
3. Link your Plex "Home" user to the new account
4. Link your Trakt.tv profile to the new account
5. Update the options for your new account (sync modes, triggers, etc..)

**Comment by jadroher** [comment]
Hello Dean



Thanks for your response. What I've done is



1 - To disconnect the trakt.tv profile - I could not disconnect the trakt profile using the configuration/accounts page (trakt-for-plex.github.io/configuration)- Pressing the X did not worked. What I could was to disconnect the Plex account from the first account configuration

2 - I created a new trakt account to be used with the plex_user account

3 - Within the configuration/accounts page I created a new erntry to be used with the plex_user

4 - I linked this new account to the new Trakt profile and the Plex_user (via home) and voila, it worked

5 - Reviewed the triggers and so



Once done I did re-linked the first account to the main plex account.



Thanks a lot for your support.

Juanjo Adroher
+34 636 841 571
jadroher@me.com

El 19 jul 2016 a las 01:57, Dean Gardiner <notifications@github.com> escribió:



@jadroher The first account on the configuration site will always sync to the administrator of your Plex Media Server, the site really needs to be updated to detail this clearly...

Try the following:

Disconnect your Trakt.tv profile from the administrator (first) account
Create a new account
Link your Plex "Home" user to the new account
Link your Trakt.tv profile to the new account
Update the options for your new account (sync modes, triggers, etc..)

**Comment by fuzeman** [comment]
@jadroher 
Glad you got it working :smile:

> I could not disconnect the trakt profile using the configuration/accounts page (trakt-for-plex.github.io/configuration)- Pressing the X did not worked.

I've got an idea what is causing this, will make sure it's fixed in a future update.

**Comment by Tanisjones** [comment]
Is it possible to link one plex account to two track users? I have tried having two rules like

MyPlexUser tracktuser1
MyPlexUser tracktuser2

but it does not work. Any ideas?

**Comment by fuzeman** [comment]
@Tanisjones Currently we only support syncing to one Trakt.tv account. I'm planning on supporting more advanced syncing configurations, but it will likely be a few months before I have time to work on this.

**Comment by almostserious** [comment]
I would like to sync my Trakt with 2 Plex accounts. 
I currently have in Plex:
The "Admin" Account
The Guest Account

In the rules i have tried:
User Rules:
Trakt Account -> ANY Plex Account => Syncs only with Admin Account
Trakt Account -> Guest Plex Account => Syncs to no account.

Any idea what else i can try? Alternatively, i'd be happy if it only syncs to Guest Account.

**Comment by fuzeman** [comment]
@almostserious I know it's not documented very well, but please note "Rules" will only apply to real-time watching activity. There is currently no way to synchronize the past watched history of two Plex accounts to one Trakt.tv account.

----------

If you want real-time watching activity for the "Guest" account to be scrobbled back to your main account, this seems to work for me:
![](https://i.imgur.com/JG8ga3Y.png)

**Note:** ratings, playback progress and collected items will only be synchronized between the Trakt.tv and Plex accounts specified on the "Accounts" page.

-------------

If you want the "Guest" account to be your primary account (with ratings being synchronized with your Trakt.tv account):

1. On the "Accounts" page in the configuration site:

    1. Unlink your Trakt.tv account from your existing "Admin" account
    2. Create a new account for the Plex "Guest" user, and link your Trakt.tv account

3. Update rules to map the "Admin" and "Guest" users to your new "Guest" account, for example:

    ![](https://i.imgur.com/4kMyI2Z.png)

**Comment by almostserious** [comment]
@fuzeman Thanks for the reply. Indeed, the scrobbling works. However, it will not sync the watched status from Trakt to the Plex Account. Also if i manually select watched status in Plex, it wlll not sync to trakt. I had the feeling this worked previously when i used only 1 account. Can you comment if this is by design?

**Comment by mbnn** [comment]
Would it be possible to delete servers not in use anymore on http://trakt-for-plex.github.io/configuration/ ? :)

**Comment by fuzeman** [comment]
@almostserious Sorry for the slow response, must of missed your comment.

> I had the feeling this worked previously when i used only 1 account. Can you comment if this is by design?

Yeah, you can have multiple Plex accounts *scrobble* back to one Trakt.tv account, but currently you can't *synchronize* multiple Plex accounts to a single Trakt.tv account (e.g. update account with past history, manually updated watched states, etc..).

Switching back to one Plex account would probably be the best solution currently.

-------------

@mbnn Have you tried deleting the servers from https://app.plex.tv/web/app#!/settings/devices? that should remove them from the connect screen as well.

**Comment by ACiDGRiM** [comment]
Not being to sync multiple Plex accounts to 1 trakt account is a little disappointing.

I am glad that the people I've shared my library with don't effect my watched status, however for ease of management I've enabled deleting media from plex with my main account and I have a secondary account I put in my livingroom that has view only incase a friend/family member is poking around where they shouldn't.

scrobbling is on multiple accounts to trakt is acceptable, however the history, manual watched status is sorely missed.

Great app by the way, I couldn't possibly make something like this but I guess it's human nature to complain about the softness of our bed.

**Comment by magicdude4eva** [comment]
Agree. We have a PlexServer where both my wife and I watch content separately. We also have a XBMC device which has the same content where we watch content together. Ideally all our history should be synced - i.e. multiple Plex accounts syncing to one trakt account.

**Comment by mbnn** [comment]
With this configuration io webpage, is it still necessary on the plugin page itself to fill in the: Authentication PIN (visit https://trakt.tv/pin/478)? Or is that only necessary when using your own account purely?

**Comment by fuzeman** [comment]
FYI, will be starting planning/development of a big syncing overhaul later this month. There will be more details available on the main issue tracker soon :smiley:

--------------

@mbnn 
> With this configuration io webpage, is it still necessary on the plugin page itself to fill in the: Authentication PIN (visit https://trakt.tv/pin/478)? Or is that only necessary when using your own account purely?

No, it's just an alternative method of authentication without requiring the configuration site.

Settings in the Plex/Web dialog are synchronized back to the plugin database, both configuration methods can be used interchangeably.

**Comment by valantislevas** [comment]
when i try to add on plex user trakt with code was ok when i try to do with basic login in plex or with plex.tv/pin for other user when i press Save , keep regenarate code to link but trakt is ok.. how i can fix that

**Comment by supremacy2k** [comment]
Got a conundrum.

1 Plex server.
2 plex.tv users.
2 trakt.tv users.

1st user (me), login to trakt and plex just fine.
2nd user (my brother) login to trakt just fine, but fails to save login details for plex. (it connects, just wont save it)

I need to sync my watched status to my trakt account, and his watched status to his trakt account.
Any advice?

Oh, and I can't remove accounts, on the accounts page under configuration.

**Comment by fuzeman** [comment]
@supremacy2k Which version of the plugin are you using? *(displayed in `Channels -> Trakt.tv -> About`)*

How are you authenticating the second Plex account? switching to "Home" mode in the Plex authentication pane may provide better results.

**Comment by supremacy2k** [comment]
@fuzeman I was using latest.
I tried with token, i tried with plain login.

As i said, loggin in was not the issue, getting the plugin to actually save it, was.

I never got a green V mark, and whenever i clicked save, it would reset, requiring me to login again.
I ended up removing the plugin. 

Sorry.

**Comment by Queo85** [comment]
Hi guys,

first of all let me thank fuzeman for this plug in. Amazing.

I'm trying to set up the accounts and rules but still haven't figure out how to do it.

I have 1 server (let's call it **QueoServer**), I then have 3 users on Plex (**GirlfriendQueo**, **Queo**, & **Sister**) and 2 Trakt accounts (**Queo** & **Sister**).

What I'm trying to match is:

- The  users GirlfriendQueo & Queo (on Plex) to sync with the Queo (Trakt) account
- Sister  (Plex) to sync with Sister (Trakt) account.


Could someone help me figure out how to do this, please?

Kind regards,
Queo

**Comment by intel352** [comment]
@fuzeman As the interface and best practices here aren't 100% clear to a lot of users it seems, perhaps it would be worthwhile to put together various scenarios in a wiki page, the configuration required for each scenario to function?

**Comment by SJ50** [comment]
@supremacy2k @fuzeman make sure user has access to particular server.
yesterday I was pulling out my hair with same issue could not get green tick for second user, turns out that I have not granted access to particular server ( I have 3 server) to the second user.
@Queo85 at this moment only each plex user to each Trackt account possible. At this multiple plex user to one trackt or on plex user to multiple trackt account is not possible.

**Comment by Ashkaan** [comment]
Can anyone help me make a User Rule for a Managed User?  None of my Managed Users are available as choices.

**Comment by Ashkaan** [comment]
I've also tried setting the Authentication user to my Managed User and Trakt to my Trakt account and then only have the "Map" rule.  Still not working.

**Comment by sixhoursago** [comment]
When I attempt to setup User Rules, not all accounts are appearing in the "Name" drop-down list. Does anyone know why this is, or how to correct it?

**Comment by Ashkaan** [comment]
It's a known bug.  I don't think anyone is actively working on it.  I have the same problem too.

**Comment by sixhoursago** [comment]
@Ashkaan Ahh gotcha. Wasn't sure we were describing the same thing. Curiously, it works as expected on my other servers.

**Comment by Ashkaan** [comment]
Interesting.  What's different about the setup that you're reporting and the working ones?

**Comment by sixhoursago** [comment]
@Ashkaan Unfortunately nothing, as far as I can determine. It's all running in Docker on identical hosts.

**Comment by sixhoursago** [comment]
@Ashkaan This issue unfucked itself in time. Some time after doing absolutely nothing about it, I found the User Rules lists to be populated as they should be.

**Comment by Ashkaan** [comment]
Mine is still quite messed up.  It scrobbles new things, but it doesn't send my collection or old watched things.

**Comment by uberchic** [comment]
I'm very new to this and i got the Trakt plugin to work for my PLEX account.  If i want to setup my subsequent users, do they need to setup their own Trakt accounts which i then input?  I want each user to keep track of their own watch/unwatched status and not have all users synced.

**Comment by sixhoursago** [comment]
@uberchic Yes they'll need to sign up for Trakt and you'll have to be communicating with them in real-time as you add their accounts to the Trakt for Plex plugin so that the verification codes can be entered as needed. 

But if you want them to keep track of their own watch/unwatched status, then you're already done. Don't bother to connect their accounts at all.

**Comment by shaddowlink** [comment]
If I want my friends in Plex, who I share libaries with, be able to sync with their trakt account, do I have to login on the configuration page into their plex accounts and put in their trakt pin (as the server owner) or is there a way they can fill it in themselves?
Just connecting to the config page gives them "unknown Error" If I allow plugins for their user, if I disable  plugins for them they get "Error 401".

**Comment by murphy52589** [comment]
I know this is way late to the party but I am stuck. I personally don't use trakt, but my friend who uses my plex server does. So I installed the plugin, followed the settings above, and his viewings still don't scrobble to trakt. I have it set to map to any and his trakt user name is the same as his plex username. We even tried with his trakt pin and it wouldn't sync. Any ideas?

**Comment by binarymelon** [comment]
@fuzeman Once the configuration is set up, can the authorization for the configuration tool be removed safely?

**Comment by pbu80** [comment]
Not able to add my friends account whom I share my library with, it says the plex account is already configured for my name.  Any ideas?

**Comment by merqwury** [comment]
Has anyone figured a way to synchronize 2 Plex accounts to 1 trakt.tv account?

**Comment by bulletproof2k** [comment]
Unable to find valid connection.
![plex-hero-1200-80](https://user-images.githubusercontent.com/1818579/194697836-a2d75364-81e9-4702-9729-6038f0f5dc58.jpg)

**Comment by bulletproof2k** [comment]
Unable to find valid connection.
![2022-10-08_11-20-17](https://user-images.githubusercontent.com/1818579/194697889-3f346809-f907-4e09-bb05-bf8a37aa5f72.png)
